clippy_lints/src/attrs.rs

```diff
@@ -121,21 +121,23 @@ impl LateLintPass for AttrPass {
                                 }
                                 if let Some(mut sugg) = snippet_opt(cx, attr.span) {
                                     if sugg.len() > 1 {
-                                        span_lint_and_then(cx, USELESS_ATTRIBUTE, attr.span,
+                                        span_lint_and_then(cx,
+                                                           USELESS_ATTRIBUTE,
+                                                           attr.span,
                                                            "useless lint attribute",
                                                            |db| {
                                             sugg.insert(1, '!');
                                             db.span_suggestion(attr.span, "if you just forgot a `!`, use", sugg);
                                         });
                                     }
                                 }
-                            },
```

Comment:
I have a strong opinion :P I really like the fact that you can distinguish a match arm block closing bracket from other closing brackets, but I can do my own PR when I get around to it.